pwiz_tools/Skyline/Test/SrmSettingsTest.cs

```diff
@@ -566,7 +566,7 @@ public void SerializeStaticModTest()
             // Bad terminus
             AssertEx.DeserializeError<StaticMod>("<static_modification name=\"Mod\" terminus=\"X\" formula=\"C23N\" />");
             // Bad formula
-            AssertEx.DeserializeError<StaticMod, ArgumentException>("<static_modification name=\"Mod\" aminoacid=\"K\" formula=\"C23NHe2\" />");
+            AssertEx.DeserializeError<StaticMod, ArgumentException>("<static_modification name=\"Mod\" aminoacid=\"K\" formula=\"C23NHx2\" />");

```

Comment:
Just ignorance on my part, but what is the significance of the change from C23NHe2 to C23NHx2?

Reply:
This test is just trying to make sure that an error would be thrown if an invalid formula was provided. It used to be that we could count on "He" being an element that Skyline did not support, since we did not have it in our list of elements. However, now that we have added all of the elements from the periodic table, I had to pick some other string that was not a real element.